### PR TITLE
Remove dependence on `Image_filename` for image selector

### PR DESCRIPTION
**Pull request by egrace479**
Displaying sample images required both `image_filename` and `file_url` columns. This was an unnecessary complication that is removed in this PR.

Also prints a more specific message when no images are displayed (indicating there are no such entries vs. there are no displayable images for those X entries). This isn't a change in options, just a change in the message itself for greater clarity.

### CODE REVIEW

**Comment by egrace479** [comment]
I also noticed a bug in the dropdown selector: if you go from specifying subspecies to selecting `Any-<species>` for the subspecies, then it will not recognize the `Any`, as it is returned at this stage as a list (`['Any']`) instead of a string (`'Any'`), which is the initial return. This is a result of the check for type that was inserted (at line 142 in the updated `components/query.py`) to avoid returning a sample image before a selection (due to dash executing the callbacks automatically once). This is something I intend to fix and update in another PR to `dev` for further testing before updating `main` and generating a new release.

File: components/query.py [hunk 1]
```diff
@@ -109,28 +109,20 @@ def get_images(df, subspecies, view, sex, hybrid, num_images):
...
-                image_path = filepaths[i] + '/' + filenames[i]
-        Imgs.append(html.Img(src = image_path, style = IMG_STYLE))
+    Imgs = [html.Img(src = filepaths[i], style = IMG_STYLE) for i in range(len(filepaths))]
+    
     return Imgs
 
 def get_filenames(df, subspecies, view, sex, hybrid, num_images):
     '''
-    Funtion to randomly select the given number of filenames for images adhering to specified filters.
+    Funtion to randomly select the given number of filepaths (file urls) for images adhering to specified filters.
```
Comment (johnbradley) [inline components/query.py:125]:
```suggestion
    Function to randomly select the given number of filepaths (file urls) for images adhering to specified filters.
```

**Review by johnbradley — APPROVED** [review]
LGTM, just one spelling fix.

File: components/query.py [hunk 2]
```diff
@@ -109,28 +109,20 @@ def get_images(df, subspecies, view, sex, hybrid, num_images):
...
-                image_path = filepaths[i] + '/' + filenames[i]
-        Imgs.append(html.Img(src = image_path, style = IMG_STYLE))
+    Imgs = [html.Img(src = filepaths[i], style = IMG_STYLE) for i in range(len(filepaths))]
+    
     return Imgs
 
 def get_filenames(df, subspecies, view, sex, hybrid, num_images):
     '''
-    Funtion to randomly select the given number of filenames for images adhering to specified filters.
+    Function to randomly select the given number of filepaths (file urls) for images adhering to specified filters.
```
Comment (hlapp) [inline components/query.py:125]:
```suggestion
    randomly selects the given number of filepaths (file urls) for images adhering to specified filters.
```

Very minor suggestion. It's a function by definition, so might as well not repeat that fact.

**Review by hlapp — APPROVED** [review]
See one nitpicking suggestion :-)

File: components/query.py (hunk 2 above)
Comment (egrace479) [reply components/query.py]: This file has many like this, and `dashboard.py` has some too. I'll standardize them all to be consistently more direct like in `graphs.py`.

**Pull request merged**